syntax = "proto3";

package echo;

import "msg.proto";

option cc_generic_services = true;

message EchoRequest {
  msg.KeyValues messages = 1;
}

message EchoResponse {
  msg.KeyValues messages = 1;
}

service EchoService {
  rpc Echo(EchoRequest) returns (EchoResponse);
}